src/components/Header/Header.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+import { SmartHRLogo } from '../SmartHRLogo/SmartHRLogo'
+import { HeaderButton } from './HeaderButton'
+import { HeaderNotification, HeaderNotificationProps } from './HeaderNotification'
+import { HeaderUserDropDown, HeaderUserDropDownProps } from './HeaderUserDropDown'
+import { HeaderEmployeeDropDown, HeaderEmployeeDropDownProps } from './HeaderEmployeeDropDown'
+
+interface Props {
+  logoUrl?: string
+  employeeListButtonLink?: string
+  isAdmin?: boolean
+  headerNotification: HeaderNotificationProps
+  headerEmployeeDropDown: HeaderEmployeeDropDownProps
+  headerUserDropDown: HeaderUserDropDownProps
```

Comment:
[IMO] Is there a need to prefix `Header component` props with `header`?
headerNotification, headerEmployeeDropDown, headerUserDropDown...